extensions/git/package.json

```diff
@@ -411,6 +411,23 @@
         "category": "Git"
       }
     ],
+    "keybindings": [
```

Comment:
Please also add mac keybindings to use `cmd` instead of `ctrl`.
Example how markdown does it
https://github.com/microsoft/vscode/blob/80cd0ee564caed53f0a7f9fe8fba5b781cbbf12d/extensions/markdown-language-features/package.json#L175